Kickstarter-iOS/Views/Cells/CommentCell.swift

```diff
@@ -0,0 +1,104 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class CommentCell: UITableViewCell, ValueCell {
+  // MARK: - Properties
+
+  private lazy var bodyTextView: UITextView = { UITextView(frame: .zero) }()
+  private lazy var bottomColumnStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var commentCellHeaderStackView: CommentCellHeaderStackView = {
+    CommentCellHeaderStackView(frame: .zero)
+  }()
+
+  private lazy var flagButton = { UIButton(frame: .zero) }()
+  private lazy var replyButton = { UIButton(frame: .zero) }()
+
+  private lazy var rootStackView = {
+    UIStackView(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private let viewModel = CommentCellViewModel()
+
+  // MARK: - Lifecycle
+
+  override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+    self.setupConstraints()
+    self.bindStyles()
+    self.configureViews()
+    self.bindViewModel()
+  }
+
+  required init?(coder aDecoder: NSCoder) {
+    super.init(coder: aDecoder)
+  }
+
+  // MARK: - Styles
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self
+      |> baseTableViewCellStyle()
+
+    _ = self.rootStackView
+      |> commentCellRootStackViewStyle
+
+    _ = self.bodyTextView
+      |> commentBodyTextViewStyle
+
+    _ = self.replyButton
+      |> replyButtonStyle
+
+    _ = self.flagButton
+      |> UIButton.lens.image(for: .normal) %~ { _ in Library.image(named: "flag") }
+  }
+
+  // MARK: - Configuration
+
+  internal func configureWith(value: (comment: Comment, viewer: User?)) {
+    self.commentCellHeaderStackView.configureWith(comment: value.comment, user: value.viewer)
+    self.viewModel.inputs.configureWith(comment: value.comment, viewer: value.viewer)
```

Comment:
Can we rename the instances where we've referred to the `user` as `viewer` to `user`, we don't have the concept of a _viewer_ in the codebase.